packages/wonder-blocks-tokens/src/tokens/color.ts

```diff
@@ -79,14 +81,18 @@ export const color = {
     // Green shades
     fadedGreen24: fadedColorWithWhite(baseColors.green, 0.24),
     fadedGreen16: fadedColorWithWhite(baseColors.green, 0.16),
+    fadedGreen8: fadedColorWithWhite(baseColors.green, 0.08),
     // Gold shades
     fadedGold24: fadedColorWithWhite(baseColors.gold, 0.24),
     fadedGold16: fadedColorWithWhite(baseColors.gold, 0.16),
+    fadedGold8: fadedColorWithWhite(baseColors.gold, 0.08),
     // Purple shades
     fadedPurple24: fadedColorWithWhite(baseColors.purple, 0.24),
     fadedPurple16: fadedColorWithWhite(baseColors.purple, 0.16),
+    fadedPurple8: fadedColorWithWhite(baseColors.purple, 0.08),
```

Comment:
**note:** New color as we agreed with design. https://www.figma.com/design/pmQkmkD4FwfIjEPVVtFn9Q?node-id=16516-1062#847262137